src/main/java/com/anonymous/privacytool/config/CustomUserDetailsService.java

```diff
@@ -0,0 +1,32 @@
+package com.anonymous.privacytool.config;
+
+import com.anonymous.privacytool.entity.User;
+import com.anonymous.privacytool.repository.AuthUserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
```

Comment:
Avoid - Some new classes/constants added in the future can lead to conflict with our code.